views/middleware.go

```diff
@@ -37,31 +38,84 @@ func (v *Views) RequiresLogin(next echo.HandlerFunc) echo.HandlerFunc {
 			}
 			return c.Redirect(http.StatusFound, "/")
 		}
-		if !userFromSession.Authenticated {
+		if !c1.User.Authenticated {
 			// Not authenticated
 			return c.Redirect(http.StatusFound, "/")
 		}
 		return next(c)
 	}
 }
 
+// RequiresLoginJSON is a middleware which will be used for each
```

Comment:
SUPER DUPER NITPICK: `is used` rather than `will be used`

Reply:
Wow, yes that is very nitpick, fixed